src/Engine/ProtoCore/ClassTable.cs

```diff
@@ -42,6 +43,27 @@ public bool IsEmpty
             }
         }
 
+        public bool IsFfiDerivedDsClass
```

Comment:
I am wondering if we really need this special property. What if we say that a DS class can be imported from FFI class only if the FFI class is marked static or all its method and properties are static? So we may think of having a property called `IsStaticClass`.